sources/packages/web/src/views/aest/institution/Locations.vue

```diff
@@ -1,13 +1,24 @@
 <template>
```

Comment:
This comment is not specific to this file, I noticed that buttons on the right side of the data table are not aligned with the table like

![image](https://user-images.githubusercontent.com/54600590/145639748-30f048ee-7455-4cb7-a5a2-9e1dc831b1b4.png)

![image](https://user-images.githubusercontent.com/54600590/145639791-3a7c09bb-cdc9-4f5e-b94f-ba1895a8ebc4.png)

![image](https://user-images.githubusercontent.com/54600590/145639842-5f025c7f-eac5-43c9-8940-aa42de234521.png)